doc/architecture/index.rst

```diff
@@ -0,0 +1,57 @@
+Architecture
+============
+
+.. panels::
+
+    :fa:`book` :doc:`consensus/index`
+    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    Different consensus protocols can be used to replicate state in a CCF network.
```

Comment:
Isn't this a little misleading as only CFT is supported for releases?

Reply:
It is, but this isn't new content, I've just moved it around.